client/stylesheets/_responsive.scss

```diff
@@ -403,6 +450,29 @@
     .profileCardLi {
         margin: 6px 5px !important;
     }
+
+    .profileChatBtn a, .optionsProClick ul .clicker {
+        font-size: 0.9rem !important;
+        padding: 12px 16px !important;
+    }
+
+    .box-title {
+        font-size: 0.9rem !important;
+    }
+
+    .optionsProClick {
+        margin-left: 5px !important;
+    }
+}
+
+@media (max-width: 589px) {
+    .profileUserGraph, .profileUserName {
+        display: none !important;
+    }
+
+    .profileUserNameHidden, .profileUserGraphHidden {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileUserNameHidden` should be written in lowercase with hyphens<br>Selector `profileUserGraphHidden` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line